src/yamlProjectConfiguration/v1.ts

```diff
@@ -17,6 +17,7 @@ enum CloudProviderIdentifier {
     SELF_HOSTED_AWS = "selfHostedAws",
     CAPYBARA = "capybara",
     CAPYBARA_LINUX = "capybaraLinux",
+    CLUSTER = "cluster",
```

Comment:
Do not update stuff in `v1.ts` as all of the file will be deprecated once all of our users update genezio to v1.0.0. 
This file here is just to still support backwards comp to users that have `genezio < v1.0.0`.

This is not the case for this feature as it lands in production only for user `genezio >= v1.xx`.